static/app/utils/profiling/renderers/flamegraphRenderer.tsx

```diff
@@ -471,17 +475,33 @@ class FlamegraphRenderer {
     this.gl.uniformMatrix3fv(this.uniforms.u_projection, false, projectionMatrix);
 
     // Model to projection
-    this.gl.uniformMatrix3fv(this.uniforms.u_model, false, configViewToPhysicalSpace);
+    this.gl.uniformMatrix3fv(
+      this.uniforms.u_model,
+      false,
+      mat3.fromValues(
+        configViewToPhysicalSpace[0],
+        configViewToPhysicalSpace[1],
+        configViewToPhysicalSpace[2],
+        configViewToPhysicalSpace[3],
+        configViewToPhysicalSpace[4],
+        configViewToPhysicalSpace[5],
+        configViewToPhysicalSpace[6],
+        configViewToPhysicalSpace[7],
+        configViewToPhysicalSpace[8]
+      )
```

Comment:
Why copy the matrix? Also, why not just `configViewToPhysicalSpace.clone()` if you need a copy?

Reply:
Oh yeah, I need up change this, was debugging the wrong X transform